src/Pact/Analyze/Types/Languages.hs

```diff
@@ -1479,6 +1495,8 @@ prettyTerm ty = \case
   GuardPasses _ _
     -> error "GuardPasses should only appear inside of an Enforce"
 
+  WithCapability a x   -> parensSep ["with-capability", pretty a, prettyTerm ty x]
+  Granting _ _ x       -> prettyTerm ty x
```

Comment:
This grants a capability after the contained term evaluates. We now inject it into the body (read: after bindings) of inlined `defcap` calls, and we will wrap `compose-capability` calls with this as well.